src/tests/ftest/io/llnl_mpi4py.yaml

```diff
@@ -0,0 +1,23 @@
+hosts:
+    test_servers:
+        - boro-A
+    test_clients:
+        - boro-B
+timeout: 240
+server:
```

Comment:
change server to server_config (Control plane patch about to merge)

Reply:
Will do.